README.md

```diff
@@ -4,45 +4,49 @@
 
 This is code associated with the paper ["GENNI: Visualising the Geometry of Equivalences for Neural Network Identifiability,"](https://drive.google.com/file/d/1mGO-rLOZ-_TXu_-8KIfSUiFEqymxs2x5/view) published in the [NeurIPS](https://nips.cc/) Workshop on [Differential Geometry meets Deep Learning 2020](https://sites.google.com/view/diffgeo4dl/).
 
-**THIS REPOSITORY WILL BE UPDATED WITH FURTHER DOCUMENTATION SOON!!**
-
 If you have any questions, please feel free to reach out to us or make an issue.
 
-## Preliminaries
+## Installing
 
-Our package is designed to run in Python 3.6 and pip version 20.2.4....
+Genni is available from PyPI [here](https://pypi.org/project/genni/). In order
+to install simply use `pip`
 
-```
-pip install -r requirements.txt
+```sh
+pip install genni
```

Comment:
this does not work...  work for me, get this error when I run 

```suggestion
python setup.py install
```

Error:

```
...
Adding genni 1.0.0 to easy-install.pth file

Installed ***/local/python3.7/site-packages/genni-1.0.0-py3.7.egg
Processing dependencies for genni==1.0.0
Searching for ray[tune]
Reading https://pypi.org/simple/ray/
Downloading https://files.pythonhosted.org/packages/b9/fb/********/ray-1.0.1.post1-***.whl#sha256=********
Best match: ray 1.0.1.post1
Processing ray-1.0.1.post1***.whl
Installing ray-1.0.1.post1-****.whl to ***/local/python3.7/site-packages
Adding ray 1.0.1.post1 to easy-install.pth file

Installed ***/local/python3.7/site-packages/***********************.egg
Traceback (most recent call last):
  File "***/local/python3.7/site-packages/pkg_resources/__init__.py", line 2748, in requires
    deps.extend(dm[safe_extra(ext)])
KeyError: 'tune'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 29, in <module>
    "tqdm",
  File "***/local/python3.7/site-packages/setuptools/__init__.py", line 144, in setup
    return distutils.core.setup(**attrs)
  File "***/local/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "***/local/python/3.7.3/***/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "***/local/python/3.7.3/***/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "***/local/python3.7/site-packages/setuptools/command/install.py", line 67, in run
    self.do_egg_install()
  File "***/local/python3.7/site-packages/setuptools/command/install.py", line 117, in do_egg_install
    cmd.run(show_deprecation=False)
  File "***/local/python3.7/site-packages/setuptools/command/easy_install.py", line 425, in run
    self.easy_install(spec, not self.no_deps)
  File "***/local/python3.7/site-packages/setuptools/command/easy_install.py", line 674, in easy_install
    return self.install_item(None, spec, tmpdir, deps, True)
  File "***/local/python3.7/site-packages/setuptools/command/easy_install.py", line 721, in install_item
    self.process_distribution(spec, dist, deps)
  File "***/local/python3.7/site-packages/setuptools/command/easy_install.py", line 766, in process_distribution
    [requirement], self.local_index, self.easy_install
  File "***/local/python3.7/site-packages/pkg_resources/__init__.py", line 795, in resolve
    new_requirements = dist.requires(req.extras)[::-1]
  File "***/local/python3.7/site-packages/pkg_resources/__init__.py", line 2751, in requires
    "%s has no such extra feature %r" % (self, ext)
pkg_resources.UnknownExtra: ray 1.0.1.post1 has no such extra feature 'tune'
```

Reply:
We don't want to do `python setup.py install` unless we can avoid it. I'll try to fix the error.

Reply:
It should work now, I've made the package have all of the dependencies needed to run by just doing pip install.

Note that I've used python=3.8 so we might want to enforce that. I know if fails for python=3.9